composer.lock

```diff
@@ -1,4239 +0,0 @@
-{
```

Comment:
This file should not have been deleted

Reply:
I just forgot to add it to the commit after deleting it during the install of a new 5.4 version of Symfony